usr/share/rear/finalize/default/060_compare_files.sh

```diff
@@ -23,15 +23,17 @@ local md5sum_stdout
 if ! md5sum_stdout="$( chroot $TARGET_FS_ROOT md5sum -c --quiet < $VAR_DIR/layout/config/files.md5sum )" ; then
     LogPrintError "Restored files in $TARGET_FS_ROOT do not fully match the recreated system"
     LogPrintError "(files in the backup are not same as when the ReaR rescue/recovery system was made)"
-    # Add two spaces indentation for better readability what the 'md5sum' stdout lines are.
     # This 'sed' call must not be done in the above command substitution as a pipe
     # because then the command substitution exit status would be the one of 'sed'.
-    # Also prefix the reported files in the 'md5sum' stdout lines with '/mnt/local'
-    # (so an implicit condition is that only files '/path/...' appear as 'md5sum' stdout lines)
+    # Prefix the reported files in the 'md5sum' stdout lines with '/mnt/local'
     # because this is the right path for the user in the currently running ReaR recovery system
     # for the restored files that do not match the md5sums that were saved at "rear mkrescue" time,
-    # cf. https://github.com/rear/rear/pull/2954#issuecomment-1467645338 and subsequent comments:
-    LogPrintError "$( sed -e "s|^|  $TARGET_FS_ROOT|" <<< "$md5sum_stdout" )"
+    # cf. https://github.com/rear/rear/pull/2954#issuecomment-1467645338 and subsequent comments.
+    # The 'md5sum' stdout lines that report files start with '/' because
+    # var/lib/rear/layout/config/files.md5sum contains file names with full path.
+    # Prefix all 'md5sum' stdout lines with two spaces indentation
+    # for better readability what the 'md5sum' stdout lines are:
+    LogPrintError "$( sed -e "s|^/|$TARGET_FS_ROOT/|" -e 's/^/  /' <<< "$md5sum_stdout" )"
     LogPrintError "Manually check if those changed files cause issues in your recreated system"
     return 1
```

Comment:
what is the actual effect of `return 1` in a ReaR script? Isn't it simply ignored? Then I'd prefer to not have those returns as it can lead other developers into thinking that it is meaningful

Reply:
I explained this already in detail at another place
which I currently do not find...

Reply:
Hmm...
I think the place where I had explained this was in
https://github.com/rear/rear/pull/2953
therein
https://github.com/rear/rear/pull/2953#discussion_r1126947469
but there is nothing there shown in GitHub.
I know I had somewhere explained this because I remember
that I had typed in an explanation plus I had searched
for the matching issues to get their links.

Reply:
I posted something without detailed explanations now here in
https://github.com/rear/rear/pull/2954#issuecomment-1471645269